pipelinesScripts/validator/validator.go

```diff
@@ -163,3 +227,15 @@ func validateContent(descriptor *utils.PluginDescriptor) error {
 	}
 	return nil
 }
+
+func runValidation(projectPath string) (err error) {
+	var output string
+	if output, err = utils.RunCommand(projectPath, true, "go", "vet", "-v", "./..."); err != nil {
+		fmt.Println("Failed to Lint plugin source code, located at " + projectPath + ". Error:\n" + output)
```

Comment:
The way we handle the messages is probably wrong. We are printing the error message using ```fmt.Println``` here, and then return the error. Then, in the calling function, we check if the error is not nil, and if not, we add another ```fmt.Println```, which basically repeats the first one. We should avoid spreading error messages all around. We should return errors, and if we need to add info to the error, we should wrap it with another.
It is okay to use ```fmt.Println``` in this flow, but only at the top of the execution flow. We can always print the value of ```err.Error()```.